anago

```diff
@@ -742,6 +743,23 @@ get_build_candidate () {
                                 $PARENT_BRANCH \
    || return 1
 
+  # When creating alphas, make sure that X.X.0 has been created first or warn.
```

Comment:
Do you mean when creating "X.Y+1.0.alpha", make sure that X.Y.0 exists?

Should we instead make sure "X.Y.0.rc.0" exists?